### DISCUSSION TITLE
openVINO AND coral + load balance

### DISCUSSION THREAD

**Discussion by jsapede** [blakeblackshear/frigate#14461] [Ask A Question] [answered]
Hello

Is it possible to have, at the same time, 2 types of detectors, say a Coral AND openVINO and to load balance some caméras on the openVINO ans others on the Coral ? How to set this in the config

**Comment by hawkeye217** [comment, marked as answer]
No, it's not possible to mix a Coral and OpenVINO.

**Reply by jsapede** [reply]
Thanks
